pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -180,14 +195,14 @@ let
       pushd out/Release/obj.target
       find . -path "./torque_*/**/*.o" -or -path "./v8*/**/*.o" | sort -u >files
       ${if stdenv.buildPlatform.isGnu then ''
-        ar -cqs $libv8/lib/libv8.a @files
+        $AR -cqs $libv8/lib/libv8.a @files
```

Comment:
Hm, I’m not sure why this line is conditional. I thought Nixpkgs bintools wrapper handled response files, but maybe that’s not the case here for some reason? I’ll have to double-check that.